inference-engine/ie_bridges/java/tests/IECoreTests.java

```diff
@@ -0,0 +1,92 @@
+import junit.framework.TestCase;
+
+import org.intel.openvino.*;
+
+import java.util.Map;
+import java.util.HashMap;
+
+public class IECoreTests extends TestCase {
+    ConfigTests config = new ConfigTests();
+
+    protected IECore core;
+    protected String exceptionMessage;
+    
+    @Override
+    protected void setUp() {
+        try {
+            System.loadLibrary(IECore.NATIVE_LIBRARY_NAME);
+        } catch (UnsatisfiedLinkError e) {
+            System.err.println("Failed to load Inference Engine library\n" + e);
+            System.exit(1);
+        }
+        core = new IECore();
+    }
+
+    @Override
+    protected void tearDown() {
+        exceptionMessage = "";
+    }
+
+    public void testInitIECore(){
+        assertTrue(core instanceof IECore);
+    }
+
+    public void testReadNetwork(){
+        CNNNetwork net = core.ReadNetwork(config.getXML(), config.getBIN());
+        assertTrue(net instanceof CNNNetwork);
```

Comment:
Let's check something more reliable. It seems to me that this conditions is always true because `net ` is defined as `CNNNetwork ` type. Let's check `net.getName()`, in example.